src/test/TestGhoGsmSteward.t.sol

```diff
@@ -0,0 +1,466 @@
+    address feeStrategy = GHO_GSM.getFeeStrategy();
+    uint256 buyFee = IGsmFeeStrategy(feeStrategy).getBuyFee(1e4);
+    uint256 sellFee = IGsmFeeStrategy(feeStrategy).getSellFee(1e4);
+    vm.prank(RISK_COUNCIL);
+    GHO_GSM_STEWARD.updateGsmBuySellFees(address(GHO_GSM), buyFee + 1, sellFee + 1);
+    IGhoGsmSteward.GsmDebounce memory timelocks = GHO_GSM_STEWARD.getGsmTimelocks(address(GHO_GSM));
+    assertEq(timelocks.gsmFeeStrategyLastUpdated, block.timestamp);
+  }
+
+  function testUpdateGsmBuySellFeesAfterTimelock() public {
+    address feeStrategy = GHO_GSM.getFeeStrategy();
+    uint256 buyFee = IGsmFeeStrategy(feeStrategy).getBuyFee(1e4);
+    uint256 sellFee = IGsmFeeStrategy(feeStrategy).getSellFee(1e4);
+    vm.prank(RISK_COUNCIL);
+    GHO_GSM_STEWARD.updateGsmBuySellFees(address(GHO_GSM), buyFee + 1, sellFee + 1);
+    skip(GHO_GSM_STEWARD.MINIMUM_DELAY() + 1);
+    uint256 newBuyFee = buyFee + 2;
+    uint256 newSellFee = sellFee + 2;
+    vm.prank(RISK_COUNCIL);
+    GHO_GSM_STEWARD.updateGsmBuySellFees(address(GHO_GSM), newBuyFee, newSellFee);
+    address newStrategy = GHO_GSM.getFeeStrategy();
+    uint256 currentBuyFee = IGsmFeeStrategy(newStrategy).getBuyFee(1e4);
+    uint256 currentSellFee = IGsmFeeStrategy(newStrategy).getSellFee(1e4);
+    assertEq(currentBuyFee, newBuyFee);
+    assertEq(currentSellFee, newSellFee);
+  }
+
+  function testUpdateGsmBuySellFeesNewStrategy() public {
+    address feeStrategy = GHO_GSM.getFeeStrategy();
+    uint256 buyFee = IGsmFeeStrategy(feeStrategy).getBuyFee(1e4);
+    uint256 sellFee = IGsmFeeStrategy(feeStrategy).getSellFee(1e4);
+    vm.prank(RISK_COUNCIL);
+    GHO_GSM_STEWARD.updateGsmBuySellFees(address(GHO_GSM), buyFee + 1, sellFee + 1);
+    address[] memory cachedStrategies = FIXED_FEE_STRATEGY_FACTORY.getFixedFeeStrategies();
+    assertEq(cachedStrategies.length, 1);
+    address newStrategy = GHO_GSM.getFeeStrategy();
+    assertEq(newStrategy, cachedStrategies[0]);
+  }
+
+  function testUpdateGsmBuySellFeesIfZeroFees() public {
+    address currentFeeStrategy = GHO_GSM.getFeeStrategy();
+    vm.mockCall(
+      currentFeeStrategy,
+      abi.encodeWithSelector(GHO_GSM_FIXED_FEE_STRATEGY.getBuyFee.selector),
+      abi.encode(0)
+    );
+    vm.mockCall(
+      currentFeeStrategy,
+      abi.encodeWithSelector(GHO_GSM_FIXED_FEE_STRATEGY.getSellFee.selector),
+      abi.encode(0)
+    );
+    uint256 buyFee = IGsmFeeStrategy(currentFeeStrategy).getBuyFee(1e4);
+    uint256 sellFee = IGsmFeeStrategy(currentFeeStrategy).getSellFee(1e4);
+    vm.prank(RISK_COUNCIL);
+    GHO_GSM_STEWARD.updateGsmBuySellFees(address(GHO_GSM), buyFee + 1, sellFee);
+    address[] memory cachedStrategies = FIXED_FEE_STRATEGY_FACTORY.getFixedFeeStrategies();
+    assertEq(cachedStrategies.length, 1);
+    address newStrategy = GHO_GSM.getFeeStrategy();
+    assertEq(newStrategy, cachedStrategies[0]);
+  }
+
+  function testRevertUpdateGsmBuySellFeesNoPreviousStrategy() public {
```

Comment:
this seems clearer to me, wdyt
```suggestion
  function testRevertUpdateGsmBuySellFeesIfZeroFeeStrategy() public {
```

Reply:
What about `testRevertUpdateGsmBuySellFeesIfZeroFeeStrategyAddress`?